### ISSUE TITLE
Allow access CIP10 methods via CLI

### ISSUE THREAD

**Issue by AlexBHarley** [celo-org/celo-monorepo#7832]
Given we have methods like this,

```
proof-of-possession --account 0x1234 --signer 5678
authorize --from 0x1234 --role vote|attestation|signer --signer 0x5678 --signature 0x9101
```

We need to remember these methods also `setIndexedSigner`. Users looking to authorise multiple signers likely shouldn't use this logic. Considering we haven't made breaking changes at the SC layer we shouldn't make them here either. I'd propose the following new methods

```
authorize-signer --from 0x1234 --role ROLE --signer 0x5678 --signature 0x9101
set-indexed-signer --from 0x1234 --role ROLE --signer 0x5678 
```

And modifying the `proof-of-possession` command to take in a `--role` parameter which will mean the resulting signature is generated with our EIP712 compatible structure.

```
proof-of-possession --account 0x1234 --signer 5678 --role ROLE

```

**Comment by nambrot** [comment]
@AlexBHarley I assume we want to do this in time CIP10 gets onto mainnet?

**Comment by nambrot** [comment]
Also ContractKit?

**Comment by AlexBHarley** [comment]
@nambrot yeah I pinged on Slack that this should be prioritised, though we have a little time until then. I have done the ContractKit changes already.

**Comment by nambrot** [comment]
Of course you did :P Feel free to pull #7854 into the sprint if that is the case. Though I'm curious who you think would interact with CIP10 on the CLI really?

**Comment by AlexBHarley** [comment]
I mean unless people are writing JS scripts themselves, there's no other way to do it. Top of mind is anyone looking to add more vote signers. I know people wanted wanted that ASAP. Will pull it in

**Comment by AlexBHarley** [comment]
@aslawson confirmed that at least one whale (Polychain) is waiting on CIP10 _and_ they use the CLI to authorise signers. I estimate this at half a day of work, the ContractKit side of it is already done. I'd highly recommend we prioritise this so it lands when CIP10 does.
